portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,34 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section" id="introduction">
+        <div class="introduction-wrapper">
+          <div>
+            <h3>Hey I'm</h3>
+            <div class="introduction-image-wrapper">
+              <img src="https://media-exp1.licdn.com/dms/image/C5603AQHvj5cLWuV6Tw/profile-displayphoto-shrink_200_200/0?e=1598486400&v=beta&t=ozRZ9uW0azRIoXdce5BafIl0VyLcY0XLbY26LwYl_GE">
```

Comment:
[please provide multimedia fallback](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)

Reply:
Thanks, corrected in commit a47d775